test/chunking-form/samples/namespace-object-import/_expected/amd/main2-3b4df12d.js

```diff
@@ -9,7 +9,7 @@ define(['exports'], function (exports) { 'use strict';
 	});
 
 	exports.a = a;
-	exports.b = b;
```

Comment:
Wow, did you implement the import pruning!? Very cool. That closes https://github.com/rollup/rollup/issues/2231 if so.